docs/open-source/livekit-webrtc.mdx

```diff
@@ -0,0 +1,45 @@
+---
+title: "Working with WebRTC"
+description: "Use LiveKit to connect Vocode Agents to WebRTC"
+---
+
+# Overview
+
+[WebRTC](https://webrtc.org/) is an alternative to websockets for real-time P2P communication. Vocode Agents are compatible with both WebRTC and websockets, enabling developers to pick
+the stack best suited for their application.
+
+To connect Vocode agents to WebRTC, Vocode uses [LiveKit](https://livekit.io/)–an open source platform for building on WebRTC. For a background on how LiveKit
+works, please see their [documentation](https://docs.livekit.io/home/get-started/intro-to-livekit/).
+
+In this guide, we'll be walking through how to connect a Vocode Agent to the [LiveKit Agents Playground](https://agents-playground.livekit.io/).
+
+# Walkthrough: hooking up a Vocode Agent to a LiveKit Room
+
+## Setting up your LiveKit Server
+
+First, you'll want to set up a LiveKit Server for your Agent. For simplicity, we are using LiveKit's hosted offering–but it can also be self hosted.
```

Comment:
```suggestion
First, you'll want to set up a LiveKit Server for your Agent. For simplicity, we are using LiveKit's hosted offering–but it can also be self hosted, since LiveKit is open source!
```